src/main/java/fi/okm/jod/yksilo/controller/profiili/ToimenkuvaController.java

```diff
@@ -46,7 +47,7 @@ List<ToimenkuvaDto> getAll(@PathVariable UUID id, @AuthenticationPrincipal JodUs
 
   @PostMapping
   @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Technical specification delta: return status code 201 instead of 200

Reply:
won't fix, spec will be changed